### PR TITLE
feat(howto): add troubleshooting documentation about fail LXD hooks

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

**Review by ru-fu — COMMENTED** [review]
I added a commit to make the entry more like the other entries - please check if everything looks good to you.

File: howto/troubleshoot.md [hunk 1]
```diff
@@ -75,3 +75,70 @@ No. The application manifest describes necessary metadata on top of the APK, whi
...
+The easiest way to make the AMS unit work again is to remove the faulty LXD node (`lxd/5` in this example) by running the following command:
+
+    juju remove-unit --force lxd/5
+
+[note type="information" status="Note"]Add `--destroy-storage` to the command if you allocated dedicated storage for LXD.[/note]
+
+After the LXD unit is successfully removed, resolve the failed hook of the AMS unit. For that, first disable
+automatic retries to prevent Juju from re-running the failed hook:
+
+    juju model-config automatically-retry-hooks=false
```
Comment (aguadoenzo) [inline howto/troubleshoot.md:133]: We should re-enable that afterwards, as some errors might go away with a few retries

**Review by aguadoenzo — CHANGES_REQUESTED** [review]

File: howto/troubleshoot.md [hunk 2]
```diff
@@ -75,3 +75,70 @@ No. The application manifest describes necessary metadata on top of the APK, whi
...
+Wait for any pending hook execution to finish. Check `juju status` to monitor the status.
+
+Once the model has settled, resolve the failed hook by running the following command:
+
+    juju resolve ams/0 --no-retry
+
+Check the `juju status` output. The status of the `ams/0` unit should switch back to `active`.
+
+To verify that the LXD cluster is still correctly in place, compare the output of `juju ssh ams/0 -- amc node ls`
+and `juju ssh lxd/0 -- lxc cluster ls`. Both commands should list the same LXD nodes.
```
Comment (ru-fu) [inline howto/troubleshoot.md:144]:
```suggestion
and `juju ssh lxd/0 -- lxc cluster ls`. Both commands should list the same LXD nodes.

Finally, enable automatic retries again:

    juju model-config automatically-retry-hooks=true

```

File: howto/troubleshoot.md (hunk 1 above)
Comment (ru-fu) [reply howto/troubleshoot.md:133]: Good catch, I added a suggestion.

**Pull request merged**